src/DropInViewer.js

```diff
@@ -43,6 +42,19 @@ export class DropInViewer extends THREE.Group {
     }
   }
 
+  /*
```

Comment:
```suggestion
  /**
```
So that IDE pick it up a documentation for the function and not just a comment. That way they can show the doc when using the function from the outside but also may use the documented argument and return type to provide autocompletion or warnings.

Reply:
updated